docs/source/tutorials/ATL03/IceSat_ATL03_Access.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>Recommend:</p><p>Or use h5glance package to interactively... -&gt; Or use the h5glance package to interactively</p><p>...keys, sub-keys and variables. -&gt; ...keys, sub-keys, and variables.</p><p>...sub-keys and allows to copy the paths of variables... -&gt; ...sub-keys and allows for the copying of paths of variables...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>